README.md

```diff
@@ -43,18 +43,39 @@ The plugin can be configured in the [**semantic-release** configuration file](ht
   "plugins": [
     "@semantic-release/commit-analyzer",
     "@semantic-release/release-notes-generator",
-    "semantic-release-hackage"
+    [
+      "semantic-release-hackage",
+      {
+        "cabalFile": "yourcabalfilename",
+        "packageName": "yourpackagename",
+        "versionPrefix": "0."
+      }
+    ]
   ]
 }
 ```
 
 ## Configuration
 
-| Property      | Description             | Default                                       |
-| ------------- | ----------------------- | --------------------------------------------- |
-| `cabalFile`   | Cabal file name         | Plugin will read your root `.cabal` file name |
-| `packageName` | Package name in Hackage |                                               |
+| Property        | Description                             | Default                                       | Required |
+| --------------- | --------------------------------------- | --------------------------------------------- | -------- |
+| `cabalFile`     | Cabal file name                         | Plugin will read your root `.cabal` file name | `false`  |
+| `packageName`   | Package name in Hackage                 |                                               | `true`   |
+| `versionPrefix` | For supporting Haskell package versions |                                               | `false`  |
 
+### Why adding a `versionPrefix` configuration?
+
+Haskell projects use a different way of versioning (A.B.C.D) than semantic versioning, you could think that since this plugin uses semantic-release, you have to move from your preview versioning way but thanks to the `versionPrefix` you could keep the usage of the 4 digits in your version.
```

Comment:
```suggestion
Haskell projects use a different versioning system (A.B.C.D) than [semantic versioning](https://semver.org/). You may think that since this is a `semantic-release` plugin, you'll have to move out from that versioning system. But thanks to the `versionPrefix` option, you can keep the 4th digit in your version.
```